src/main/scala/org/broadinstitute/dsde/firecloud/webservice/LibraryApiService.scala

```diff
@@ -79,8 +81,28 @@ trait LibraryApiService extends HttpService with FireCloudRequestBuilding
                 LibraryService.IndexAll)
             }
           }
+        } ~
+        pathPrefix("libraries") {
```

Comment:
this should probably be `pathEnd` or `pathEndOrSingleSlash` (see http://spray.io/documentation/1.2.4/spray-routing/predefined-directives-alphabetically/), to be precise about path-matching.

Please ignore (or fix!) all the other routes that do this wrong :(